src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/model/GetFeed.java

```diff
@@ -16,10 +19,14 @@
 
     private LanternMainActivity activity;
     private String proxyAddr = "";
+    private ProgressBar progressBar;
 
     public GetFeed(LanternMainActivity activity, String proxyAddr) {
         this.activity = activity;
         this.proxyAddr = proxyAddr;
+        progressBar = (ProgressBar)activity.findViewById(R.id.progressBar);
+        // show progress bar
+        progressBar.setVisibility(View.VISIBLE);
```

Comment:
FWIW, doing this in preExecute() makes sure it runs on the UI thread no matter how/when GetFeed is instantiated.